### DISCUSSION TITLE
How to provide custom example value for single return

### DISCUSSION THREAD

**Discussion by retnikt** [fastapi/fastapi#7574] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I provide a custom example value for the return value of routes? When the return is a Pydantic model, you can set the `example` keyword argument on a pydantic `Field` or add it to `schema_extra` in the Model config. However, when the return value is a simple type such as `str` or List[Optional[int]]` there seems to be no way to provide an example. Am I missing something, or is this a missing feature?

**Comment by phy25** [comment]
https://github.com/tiangolo/fastapi/issues/1007#issuecomment-589029073

**Comment by retnikt** [comment]
@phy25 I am asking about *responses* not request parameters.

**Comment by chris-allnutt** [comment]
you can manually extend/alter the returned schema, but I'd think there would be a way to do this
https://fastapi.tiangolo.com/advanced/extending-openapi/

**Comment by nkhitrov** [comment, marked as answer]
Hi, @retnikt ! 

You can add examples for in field `responses`. See [this page](https://fastapi.tiangolo.com/advanced/additional-responses/#combining-information)

**Comment by retnikt** [comment]
Thanks @Slyfoxy

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

Thanks for reporting back and closing the issue @retnikt  :+1:
